### ISSUE TITLE
Error exporting Things to csv

### ISSUE THREAD

**Issue by jeredactyl** [thingsapi/things-cli#16]
**To Reproduce**
1. Run `things-cli --csv --recursive anytime > all.csv && open all.csv` at Terminal

**Expected behavior**
all.csv is created

**Additional context**
```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/bin/things-cli", line 33, in <module>
    sys.exit(load_entry_point('things-cli==0.1.4', 'console_scripts', 'things-cli')())
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things_cli/cli.py", line 487, in main
    ThingsCLI().main()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things_cli/cli.py", line 406, in main
    self.main(ThingsCLI.get_parser().parse_args())
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things_cli/cli.py", line 422, in main
    self.parse_command(defaults, args)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things_cli/cli.py", line 479, in parse_command
    self.print_tasks(getattr(api, command)(**defaults))
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things/api.py", line 556, in anytime
    return tasks(start="Anytime", **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things/api.py", line 190, in tasks
    database = pop_database(kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things/api.py", line 719, in pop_database
    database = Database(filepath=filepath, print_sql=print_sql)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/things/database.py", line 164, in __init__
    with open(self.filepath) as file:
NotADirectoryError: [Errno 20] Not a directory: '/Users/jbachmann/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/Things Database.thingsdatabase/main.sqlite'
```

**Comment by patrick91** [comment]
passing the new db using `--database path` might work, but I get an issue of a missing table (they might have changed that too)

**Comment by patrick91** [comment]
Fixed https://github.com/thingsapi/things.py/pull/103

**Comment by ishan1m** [comment]
@patrick91 Is there a way for me to get your fix and use it? Sorry for the naive question - not quite sure the relation between things-cli and thingsapi etc

**Comment by AlexanderWillner** [comment]
Fixed. Just released Version 0.2.0
